moxy/org.eclipse.persistence.moxy/src/test/java/org/eclipse/persistence/testing/jaxb/xmlmarshaller/UnmarshallValidationTestCases.java

```diff
@@ -59,7 +53,7 @@ public void setUp() throws Exception {
         contextPath = System.getProperty("jaxb.test.contextpath", JAXBSAXTestSuite.CONTEXT_PATH);
 
         JAXBContext context = JAXBContext.newInstance(contextPath);
-        unmarshaller = context.createUnmarshaller();
+        unmarshaller = (JAXBUnmarshaller) context.createUnmarshaller();
```

Comment:
cast should not be needed

Reply:
Validation related methods were removed from API so cast to moxy implementing class is needed here.

Reply:
correct, my bad